MediaBrowser.WebDashboard/Api/DashboardService.cs

```diff
@@ -150,7 +150,7 @@ public string DashboardUIPath
                     return _serverConfigurationManager.Configuration.DashboardSourcePath;
                 }
 
-                return Path.Combine(_serverConfigurationManager.ApplicationPaths.ApplicationResourcesPath, "dashboard-ui");
+                return Path.Combine(_serverConfigurationManager.ApplicationPaths.ApplicationResourcesPath, "jellyfin-web/src");
```

Comment:
Are you sure this is correct? I don't see `src` subfolder in the submodule repo at all, but I see `dashboard-ui` in there.